### DISCUSSION TITLE
Integrate EarTrumpet into Microsoft PowerToys

### DISCUSSION THREAD

**Discussion by pbrandstetter** [File-New-Project/EarTrumpet#749] [Ideas]
Integrating EarTrumpet into PowerToys is a long wanted feature. Since EarTrumpet already uses Microsofts UI/UX and works perfectly fine in my opinion, merging both projects would be really cool.

## Related Issues from [microsoft/PowerToys](https://github.com/microsoft/PowerToys)

[Volume Control & Mute/Un-Mute In Window Titlebar](https://github.com/microsoft/PowerToys/issues/6388)
[Sound mixer as a toy](https://github.com/microsoft/PowerToys/issues/4964)

I hope you like that idea and give it a try 😊

**Comment by riverar** [comment]
Thanks, Clint and I definitely talk behind the scenes.
